src/clims/api/serializers/models/work_batch_details.py

```diff
@@ -8,6 +8,14 @@ class WorkBatchDetailsSerializer(serializers.Serializer):
     id = serializers.IntegerField(read_only=True)
     name = serializers.CharField(read_only=True)
     properties = DictField(child=ExtensiblePropertySerializer(), allow_null=True)
+    created_at = serializers.DateTimeField(read_only=True)
+    updated_at = serializers.DateTimeField(read_only=True)
+
+    # These are still mocked
+    transitions = serializers.SerializerMethodField(read_only=True)
```

Comment:
All of this stuff I'll revisit next. It was convenient for me to keep it in this PR.